x-pack/platform/plugins/shared/streams_app/public/plugin.tsx

```diff
@@ -58,6 +58,7 @@ export const renderApp = ({
 
   const appWrapperClassName = css`
     overflow: auto;
+    height: 0;
```

Comment:
don't ask me why, but it works (I think?) - my mental model is that this is setting the height to 0, but flex layout is making it grow up to the max-height, but not more. If the `height` is bigger, it won't get smaller than that.